app/workers/subject_metadata_worker.rb

```diff
@@ -0,0 +1,24 @@
+class SubjectMetadataWorker
+  include Sidekiq::Worker
+
+  def perform(set_id)
+
+    smses = SetMemberSubject.where(subject_set_id: set_id)
+
+    update_sms_priority_sql = <<-SQL
+      UPDATE set_member_subjects
+      SET    priority = CAST(subjects.metadata->>'#priority' AS INTEGER)
+      FROM   subjects
+      WHERE  subjects.id = set_member_subjects.subject_id
```

Comment:
I think we need a where clause to ensure we only attempt to cast existing priority values as well.